commands/auth.go

```diff
@@ -247,7 +247,7 @@ func loginCmdF(cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	fmt.Printf("\n  credentials for %q: \"%s@%s\" stored\n\n", name, username, url)
+	printer.Print(fmt.Sprintf("\n  credentials for %q: \"%s@%s\" stored\n", name, username, url))
```

Comment:
Wondering how much work it would be to add a `printer.Printf` interface method as well, (to avoid Sprinting this into the same method)? It's out of scope for this PR, and we can totally create a separate HW ticket if you prefer. Just wanted to get an idea of the effort.

Reply:
Hmm, the printer in mmctl is more `text/template` oriented, as the value is being used in a template always. If you are thinking adding another method as `printer.Printf(format string, a ...any)`, I think it would be pretty easy if we just execute a fmt.Sprintf and use printer.Print function. Ie:
```Go
func Printf(format string, a ...interface{}) {
	s := fmt.Sprintf(format, a...)
	Print(s)
}
```
The reason I didn't add this was it was not sharing the same mindset with `printer.Print` as `fmt.Print` does with `fmt.Printf` etc.

Reply:
I see, okay then I don't see a need. This is fine as it is.